download.jl

```diff
@@ -0,0 +1,82 @@
+#Extracting wind data from MET file 
+download("https://earthsciml.s3.amazonaws.com/compressed_Input/met/mcipv5/METDOT3D_160701.bson")
```

Comment:
The `download` function returns the location to the downloaded file, so you can just say `file = donwload(...)` and then delete the `file=` line below.